control/reporter_test.go

```diff
@@ -0,0 +1,31 @@
+package control
```

Comment:
Since this test suite isn't accessing private identifiers, should we use the `control_test` package?